test/raw_data_gis/scripts/test_us_eco.json

```diff
@@ -5,7 +5,6 @@
   "format": "vector",
   "spatial_ref": "NAD83",
   "citation": "N/A",
-  "license": [],
```

Comment:
If no license is to provided, it must simply not be mentioned, providing an empty list results in error when any operation like `citation` is done.